nixos/modules/services/home-automation/zwave-js-server.nix

```diff
@@ -0,0 +1,139 @@
+{config, pkgs, lib, ...}:
+
+with lib;
+
+let
+  cfg = config.services.zwave-js-server;
+  cacheDir = "/var/cache/zwave-js-server";
+  runtimeDir = "/run/zwave-js-server";
+  defaultConfig = {
+    storage = {
+      inherit cacheDir;
+    };
+  };
+  mergedConfigFile = "${runtimeDir}/config.json";
+in {
+
+  options.services.zwave-js-server = {
+    enable = mkEnableOption (mdDoc "Start the zwave-js-server on boot");
+
+    package = mkOption {
+      type = types.package;
+      default = pkgs.nodePackages."@zwave-js/server";
```

Comment:
Needs `defaultText = literalExpression expr` for the manual error to go away.

Also can we convert the package to `buildNpmPackage`? It is barely maintainable like this.

Reply:
Yep, just fixed the defaultText.

I didn't look at the package at all since it "just worked", I could have a quick look though.

Reply:
Is `buildNpmPackage` actually used? I could only find a couple of hits in nixpkgs, unless I'm missing something?

Reply:
Yes, it is new and shiny, but gets used more and more every day.

Reply:
This worked fairly well (apart from the node-js module for some reason declaring dependencies it needs as dev dependencies!), thanks.